src/wp-admin/css/forms.css

```diff
@@ -101,8 +101,8 @@ input[type="checkbox"]:focus,
 input[type="radio"]:focus,
 select:focus,
 textarea:focus {
-	border-color: #2271b1;
-	box-shadow: 0 0 0 1px #2271b1;
+	border-color: var(--wp-admin--form-fields--border--focus);
```

Comment:
@danfarrow is there a reason for separating `--wp-admin--form-fields--border--focus` and `--wp-admin-theme-primary` in this selector declaration?